### ISSUE TITLE
I am the author of CAM

### ISSUE THREAD

**Issue by zhoubolei** [hiveml/tensorflow-grad-cam#2]
Thanks first for introducing this awesome tensorflow-grad-cam and this blog article (https://thehive.ai/blog/inside-a-neural-networks-mind). 

The overview on the CAM and grad-CAM in the blog article is great, but somehow I found there is some bias or misleading claim to CAM, compared to grad-CAM. I would like to clarify that below:

First of all, nowadays all the mainstream network architectures such as resnet, densenet, or other squeezenet use global average pooling at the end, so the class activation map (the heatmap) could be generated directly using CAM, without modifying any network architectures. So the claim that the grad-Cam is superior over CAM because of using grad-cam without modifying architecture is false. 

Meanwhile, in your demo code, you are already using the resnet, so you can basically generate heatmap using CAM directly (see example code at https://github.com/metalbubble/CAM/blob/master/pytorch_CAM.py), without needing the extra step to compute the gradient . Through that you save the backward computation, you save almost half of the computation. This is crucial in some application such as video processing that all the need is the forward pass to generate the prediction and heatmap for each frame. 

In your demo code, you can generate the heatmaps using CAM and grad-CAM, so you can compare the difference.

**Comment by ryanjay0** [comment]
Thanks for your input. I'm honored to be contacted by the creator of CAM. I agree with the comments you made and i'll revise the blog to be more accurate.
